x-pack/plugins/apm/server/lib/alerts/action_variables.ts

```diff
@@ -45,4 +45,24 @@ export const apmActionVariables = {
     ),
     name: 'triggerValue',
   },
+  windowSize: {
+    description: i18n.translate(
+      'xpack.apm.alerts.action_variables.intervalSize',
+      {
+        defaultMessage:
+          'The interval size the alert will use to check for any incident',
+      }
+    ),
+    name: 'intervalSize',
+  },
+  windowUnit: {
+    description: i18n.translate(
+      'xpack.apm.alerts.action_variables.intervalUnit',
+      {
+        defaultMessage:
+          'The interval unit the alert will use to check for any incident',
```

Comment:
Why do we split the interval value and unit? This is only for the template, right? Unless they can do calculations it's just a label so would simplify it to have value and unit in a single variable

Reply:
that sounds good, I'll leave only the `interval` property and in the description, I'll say that it comes with the unit too.